### PR TITLE
Fix spelling mistakes

### PR DESCRIPTION
**Pull request by dpt**

### CODE REVIEW

**Comment by dpt** [comment]
Thanks for the great site. I thought I'd contribute some spelling mistake fixes spotted while reading.

It was unclear to me whether it was British or American spelling so I've left all of those as-is.

**Comment by Pomax** [comment]
Thanks! And: absolutely British =)

File: components/sections/abc/content.en-GB.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ And that's it, all values found.
 
 <div className="note">
 
-Much like the *u(t)* function in the above note, the *ratio(t)* function depends on whether we're looking at quadratic or cubic curves. Their form is intrinsically related to the *u(t)* function in that they both come rolling out of the same function evalution, explained over on [MathOverflow](http://mathoverflow.net/questions/122257/finding-the-formula-for-Bézier-curve-ratios-hull-point-point-baseline) by Boris Zbarsky and myself. The ratio functions are the "s(t)" functions from the answers there, while the "u(t)" functions have the same name both here and on MathOverflow.
+Much like the *u(t)* function in the above note, the *ratio(t)* function depends on whether we're looking at quadratic or cubic curves. Their form is intrinsically related to the *u(t)* function in that they both come rolling out of the same function evaluation, explained over on [MathOverflow](http://mathoverflow.net/questions/122257/finding-the-formula-for-Bézier-curve-ratios-hull-point-point-baseline) by Boris Zbarsky and myself. The ratio functions are the "s(t)" functions from the answers there, while the "u(t)" functions have the same name both here and on MathOverflow.
```
Comment (Pomax) [inline components/sections/abc/content.en-GB.md]: I have no idea what github is saying here - there does not appear to be a difference.

File: components/sections/arcapproximation/content.en-GB.md [hunk 2]
```diff
@@ -46,4 +46,4 @@ With that in place, all that's left now is to "restart" the procedure by treatin
...
+So... what is this good for? Obviously, If you're working with technologies that can't do curves, but can do lines and circles, then the answer is pretty straight-forward, but what else? There are some reasons why you might need this technique: using circular arcs means you can determine whether a coordinate lies "on" your curve really easily: simply compute the distance to each circular arc center, and if any of those are close to the arc radii, at an angle between the arc start and end: bingo, this point can be treated as lying "on the curve". Another benefit is that this approximation is "linear": you can almost trivially travel along the arcs at fixed speed. You can also trivially compute the arc length of the approximated curve (it's a bit like curve flattening). The only thing to bear in mind is that this is a lossy equivalence: things that you compute based on the approximation are guaranteed "off" by some small value, and depending on how much precision you need, arc approximation is either going to be super useful, or completely useless. It's up to you to decide which, based on your application!
```
Comment (Pomax) [inline components/sections/arcapproximation/content.en-GB.md]: again, there does not appear to be a difference... very odd

File: components/sections/abc/content.en-GB.md (hunk 1 above)
Comment (dpt) [reply components/sections/abc/content.en-GB.md]: That one's evalution -> evaluation

It's easier to see in github's "rich diff".

File: components/sections/arcapproximation/content.en-GB.md (hunk 2 above)
Comment (dpt) [reply components/sections/arcapproximation/content.en-GB.md]: betwee -> between in that one.

File: components/sections/offsetting/content.en-GB.md [hunk 3]
```diff
@@ -47,13 +47,13 @@ And that's where things go wrong. It doesn't even really matter what the second
...
 
-So, since the tangent length isn't a polynomial, the normalised tangent won't be a polynomial either, which means `N(t)` won't be a polynomial, which means that `d` times `N(t)` won't be a polynomial, which means that, ultimately, `O(t)` won't be a polynomial, which means that even if we can determine the function for `O(t)` just fine (and that's far from trivial!), it simply cannot be represented as a Bézier curve.
+So, since the tangent length isn't a polynomial, the normalized tangent won't be a polynomial either, which means `N(t)` won't be a polynomial, which means that `d` times `N(t)` won't be a polynomial, which means that, ultimately, `O(t)` won't be a polynomial, which means that even if we can determine the function for `O(t)` just fine (and that's far from trivial!), it simply cannot be represented as a Bézier curve.
```
Comment (Pomax) [inline components/sections/offsetting/content.en-GB.md]: both are valid in British spelling, and as such please revert this to using `s` =)

File: components/sections/abc/content.en-GB.md (hunk 1 above)
Comment (Pomax) [reply components/sections/abc/content.en-GB.md]: Where would I find github's "rich diff"? I only know about the unified vs. split view, and neither of those shows the actual different.

Comment (dpt) [reply components/sections/abc/content.en-GB.md]: Do you see a document icon in the toolbar above each diff? That's the rich diff button (yeah, it's not obvious).

Comment (Pomax) [reply components/sections/abc/content.en-GB.md]: dear lord, you need to click that for every file? That's kind of ridiculous. Thanks for pointing out it exists at least.

**Comment by dpt** [comment]
Reverted that one letter as requested.

**Comment by Pomax** [comment]
awesome, thanks!

**Pull request merged**